### ISSUE TITLE
[BUG] Managed Speed doesn't work on approach

### ISSUE THREAD

**Issue by christwigger** [flybywiresim/a32nx#3804]
**Mod Version**
Custom FBW via installer

**Describe the bug**
When engaging managed speed after being in selected speed on on approach aircraft tried to accelerate to 250 knots regardless of flap/gear settings

**To Reproduce**
1. Set speed to 160 knots on approach with gear down and full flaps
2. Engage managed speed

**Expected behavior**
Aircraft should maintain the appropriate speed for the flap levels

**Actual behavior**
Aircraft attempts to accelerate to 250 knots.

**Additional context**
I have only had this occur after being asked to maintain a certain speed e.g. 160 until 4 DME. When engaging managed speed once again the aircraft attempts to accelerate to 250 knots. This has happened on both flights I've performed today.

Was this working before/when did the issue start occurring?
This was working yesterday prior to latest update (04 March) to the custom FBW via installer

Is this a problem in the vanilla unmodded game?
No

Discord username (if different from GitHub): Twiggy145

**Comment by MisterChocker** [comment]
Have you checked the performance page to ensure you are in approach phase?

**Comment by christwigger** [comment]
Affirmative. Aircraft was in approach mode. Doing another flight now. Will report back if I encounter the bug again.

**Comment by christwigger** [comment]
Couldn't replicate on this approach. Perhaps I didn't engage approach mode. I'll keep an eye out for this "bug" in future flights and report back. I fly several times a day so should have something by tomorrow (Saturday) morning.

**Comment by christwigger** [comment]
It seems this was my error. Please close. 

Thanks Anyway.

**Comment by gizmo71** [comment]
This has happened to me too, earlier this week, on the custom FBW branch. Selected approach phase in the MCDU with the speed in selected mode on the FCU (somewhere around 210/220 knots). On pushing the speed selector to go back to managed, aircraft accelerated back towards 250 knots; reverted to selected speed, selected flaps 1, back to managed speed and all happy again.

**Comment by MisterChocker** [comment]
> This has happened to me too, earlier this week, on the custom FBW branch. Selected approach phase in the MCDU with the speed in selected mode on the FCU (somewhere around 210/220 knots). On pushing the speed selector to go back to managed, aircraft accelerated back towards 250 knots; reverted to selected speed, selected flaps 1, back to managed speed and all happy again.

@gizmo71

Interesting, you haven't filed a bug report yet, haven't you?

can you please post the content of the build_info.json

**Comment by MisterChocker** [comment]
Duplicate of #2463

**Comment by MisterChocker** [comment]
My mistake, wasn't intended to close.

**Comment by gizmo71** [comment]
> Interesting, you haven't filed a bug report yet, haven't you?

No, I stumbled onto this report having been unable to reproduce what I'd seen.

> can you please post the content of the build_info.json

I've updated a few times since it happened so it won't be right now; I've had a couple of goes at reproducing it, including activating the approach phase in the climb and cruise phases (the latter was what I did when it happened) but haven't seen the issue again.

If I do see it again I will certainly post more details. Hopefully the custom AP will render this issue moot.
